app/src/main/java/org/wikipedia/settings/RemoteConfig.kt

```diff
@@ -0,0 +1,31 @@
+package org.wikipedia.settings
+
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Store for config values that are retrieved from a server,
+ * and refreshed periodically.
+ */
```

Comment:
Let us get rid of such not-so-helpful comments since we have adapted self-commenting code.

Reply:
Done.